docs/conf.py

```diff
@@ -121,4 +121,4 @@
 # nbsphinx
 nbsphinx_timeout = 600
 napoleon_use_ivar = True
-nbsphinx_allow_errors = True # TODO: remove once notebooks final
+nbsphinx_allow_errors = True
```

Comment:
This needs to be fixed / removed.

Reply:
ok I've set this to `False` just to have it as reference for future building of the docs. I've built the docs after setting to `False` and confirm they still build.